EIPS/eip-2003.md

```diff
@@ -0,0 +1,108 @@
+---
+eip: 2003
+title: EVMC modules for implementations of precompiled contracts
+author: Paweł Bylica (@chfast)
+discussions-to: HERE
+status: Draft
+type: Informational
+created: 2019-05-09
+requires: 1352
+---
+
+## Abstract
+
+[EVMC] specifies a generic API for Ethereum execution engines.
+This EIP specifies a way of providing implementations of Ethereum precompiled contracts
+using the [EVMC VM API].
+
+## Specification
```

Comment:
I think we need to split this section into two: "Precompile considerations" and "Client considerations".

Would like to propose that `evmc_context` can be NULL for precompiles.

Reply:
Make sense